src/validations/passport-gated/index.ts

```diff
@@ -167,8 +167,10 @@ export default class extends Validation {
   async validate(currentAddress = this.author): Promise<boolean> {
     const requiredStamps = this.params.stamps || [];
     const operator = this.params.operator;
-    const scoreThreshold = this.params.scoreThreshold || 0;
-    if (!operator) throw new Error('Operator is required');
+    const scoreThreshold = this.params.scoreThreshold;
+
+    if (scoreThreshold === undefined) throw new Error('Score threshold is required');
+    if (requiredStamps.length > 0 && !operator) throw new Error('Operator is required');
```

Comment:
Maybe `Operator is required when selecting required stamps`

Reply:
Something we can definitely consider, but wouldn't consider that a blocker for this work. What are some next steps we can take to merge this one?

Reply:
```suggestion
    if (requiredStamps.length > 0 && !operator) throw new Error('Operator is required when selecting required stamps');
```

Reply:
I will change this in a separate PR (no access to add commit to this PR)